spark-jobs/src/test/scala/za/co/absa/enceladus/conformance/interpreter/rules/FillNullsRuleSuite.scala

```diff
@@ -40,8 +40,17 @@ class FillNullsRuleSuite extends FunSuite with SparkTestBase with TestRuleBehavi
     value = "Gshj1"
   )
 
+  private val literalDateRule = FillNullsConformanceRule(
+    order = 2,
+    outputColumn = "date2",
+    controlCheckpoint = false,
+    inputColumn = "date",
+    value = "1900-05-05"
+  )
+
   private val literalRulesList1 = List(literalRule)
   private val literalRulesList2 = List(literalRule, literalArrayRule)
+  private val literalRulesList3 = List(literalRule, literalArrayRule, literalDateRule)
```

Comment:
**Tiny:** You changed the test names not to refer to literal. I think these `val`s suffer from same. But they are private...

Reply:
Done